modules/observe/adoption-insights/proc-customize-adoption-insights.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * assemblies/assembly-rhdh-observability.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="proc-customize-adoption-insights_{context}"]
+= Customizing the Adoption Insights plugin in {product}
+
+You can customize the Adoption Insights plugin to suit your needs by disabling or re-enabling it, and adjusting other settings as needed.
+
+For the {product} Adoption Insights plugin, you must manually disable the plugin.
+
+.Procedure
+
+* To customize `maxBufferSize`, `flushInterval`, `debug`, and `licensedUsers` in the Adoption Insights plugin, in your {product} `app-config.yaml` file, update the relevant settings as shown in the following code:
++
+.`app-config.yaml` fragment
+[source,terminal]
+----
+app:
+  analytics:
+    adoptionInsights: 
+      maxBufferSize: _<maximum_buffer_size>_ <1>
+      flushInterval: _<flush_interval>_ <2>
+      debug: _<debug_value>_ <3>
+      licensedUsers: _<licensed_users>_ <4>
+----
+<1> (Optional) Specifies the maximum buffer size for event batching. The default value is `20`. 
+<2> (Optional) Specifies the flush interval in milliseconds for event batching. The default value is `5000ms`.
+<3> (Optional) The default value is `false`.
+<4> (Optional) Specifies the maximum number of licensed users who can access the RHDH instance. The default value is `100`.
+
+* To disable the Adoption Insights plugin, in your `app-config-dynamic.yaml` file, update the `package.disabled` value to `false` as shown in the following example:
++
+[source,yaml]
+----
+  - package: oci://ghcr.io/redhat-developer/rhdh-plugin-export-overlays/red-hat-developer-hub-backstage-plugin-adoption-insights:bs_1.35.1__0.0.3!red-hat-developer-hub-backstage-plugin-adoption-insights
+    disabled: true
+    pluginConfig:
+      dynamicPlugins:
+        frontend:
+          red-hat-developer-hub.backstage-plugin-analytics-module-adoption-insights:
+            analyticsApiExtensions:
+              - importName: AdoptionInsightsAnalyticsApi
```

Comment:
From 1.7 onwards, we no longer need to use oci images. The plugins are bundled so we can disable all three adoption insights plugins using below config.

```suggestion
  - package: ./dynamic-plugins/dist/red-hat-developer-hub-backstage-plugin-adoption-insights
    disabled: true
  - package: ./dynamic-plugins/dist/red-hat-developer-hub-backstage-plugin-adoption-insights-backend-dynamic
    disabled: true
  - package: ./dynamic-plugins/dist/red-hat-developer-hub-backstage-plugin-analytics-module-adoption-insights-dynamic
    disabled: true
```

Reply:
Also, If some users are already using ghcr images to load adoption insights plugins based on our [1.5 documentation](https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.5/html-single/adoption_insights_in_red_hat_developer_hub/index#proc-install-adoption-insights_title-adoption-insights), if they upgrade to 1.7 they no longer need that old configuration as 1.7 ships and enable these plugins by default, so they can safely remove that configurations altogether.

Calling this out somewhere in documentation would help upgrading users, wdyt?